client/app/account/user/credentials/user-credentials.service.js

```diff
@@ -0,0 +1,40 @@
+angular.module("UserAccount.services")
+    .service("UseraccountCredentialsService", class UseraccountCredentialsService {
+        constructor (OvhHttp) {
+            this.OvhHttp = OvhHttp;
+        }
+
+        getCredentials () {
+            return this.OvhHttp.get("/me/api/credential", {
```

Comment:
you should use a variable for the pattern: "/me/api/credential" and use it across your methods :)

Reply:
`OvhApiMe` will be used.